.github/workflows/playwright-visual-regression.yml

```diff
@@ -42,8 +42,7 @@ jobs:
           npx playwright test tests/visual-regression/components/ --project=tests-chromium
 
       - uses: actions/upload-artifact@v4
-        if: failure()
```

Comment:
does removing this mean we are now collecting test running reports not only upon test failure, but every test run?

Reply:
I did this to test if reports were being generated and saved into artifacts since not many runs end up in failures. Good catch, I'll restate this.